### DISCUSSION TITLE
Cannot run XCode

### DISCUSSION THREAD

**Discussion by mantoljak** [MessageKit/MessageKit#1767] [Q&A]
I have a problem with XCode running. It constantly cannot build while my code is good. Something with conversion is the problem but i cannot find it. Please help I cannot continue to work. Xcode was able to run before even with this error, but when I reopened the project, now I cannot run it at all. There is nothing red underlined in the code.

Xcode Error: Cannot convert return expression of type '()' to return type 'Message'

[Files with Error.zip](https://github.com/MessageKit/MessageKit/files/10189012/Files.with.Error.zip)

**Comment by Kaspik** [comment]
Not sure what is `private lazy var messages = [Message]()` - it should be simply `private var messages: [Message] = []`?
